social_media/migrations/0007_postsaved_postsaved_unique_post_saved.py

```
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.23 on 2025-07-30 16:58
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('social_media', '0006_postcommentlike_postcommentlike_unique_comment_like'),
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)